src/plugins/navigation/common/constants.ts

```diff
@@ -6,6 +6,6 @@
  * Side Public License, v 1.
  */
 
-export const SOLUTION_NAV_FEATURE_FLAG_NAME = 'navigation.solutionNavEnabled';
+export const SOLUTION_NAV_FEATURE_FLAG_NAME = 'solutionNavEnabled';
```

Comment:
is this intentional?

Reply:
Yes, I am using this same PR to set the correct LD key and make it available in QA